features/messages/impl/src/test/kotlin/io/element/android/features/messages/impl/timeline/TimelinePresenterTest.kt

```diff
@@ -121,7 +121,7 @@ class TimelinePresenterTest {
     fun `present - on scroll finished send read receipt if an event is before the index`() = runTest {
         val timeline = FakeMatrixTimeline(
             initialTimelineItems = listOf(
-                MatrixTimelineItem.Event(0, anEventTimelineItem())
+                MatrixTimelineItem.Event("0", anEventTimelineItem())
```

Comment:
I understand that we do not really care about the id in test. So maybe create a `FAKE_ID` const and use it every where, to replace `"0"` and `"0L"`.

Reply:
Yes thats a good idea.